e2e/tests/core/02-client/client_test.go

```diff
@@ -83,7 +83,7 @@ func (s *ClientTestSuite) TestScheduleIBCUpgrade_Succeeds() {
 	chainA, chainB := s.GetChains()
 	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
 
-	const planHeight = int64(75)
```

Comment:
does this need to change to 300?

Reply:
whoops sorry I missed this comment. 😓 

With the addition of the polling in interchain test, it appears  as if the upgrade was being scheduled in the past 

`failed to schedule upgrade: upgrade cannot be scheduled in the past:`

It looks like the polling extends the duration a bit.